bokeh/util/serialization.py

```diff
@@ -150,6 +150,10 @@ def transform_array_to_list(array):
         transformed[np.isposinf(array)] = 'Infinity'
         transformed[np.isneginf(array)] = '-Infinity'
         return transformed.tolist()
+    elif (array.dtype.kind == 'O' and pd.isnull(array).any()):
```

Comment:
if the problem is that there are actual pandas nulls then the solution is just to and `and pd` to the if above I think? Can't have pandas nulls without pandas present.